Formula/docker-machine-driver-xhyve.rb

```diff
@@ -23,6 +23,8 @@ class DockerMachineDriverXhyve < Formula
     depends_on "libev"
   end
 
+  patch :DATA
```

Comment:
Ah, In other words, I will not this patch to my repository, because have side effects.
This patch is Homebrew specific.

I'll add `# Allow specifying version and libev location`.